.appveyor.yml

```diff
@@ -51,6 +51,8 @@ test_script:
   # to put the Python version you want to use on PATH.
   - |
     %PYTHON%\Scripts\tox.exe -e coverage
+    %PYTHON%\Scripts\tox.exe -e pyinstaller
+    dist\Tahoe-LAFS\tahoe.exe --version
```

Comment:
Probably worth a comment here just pointing out that this is a minimal proof that the pyinstaller thing worked.  We don't really care about the version, we care that we can run the exe without exploding on some random pkg_resources error or ImportError.

Reply:
Good call; thanks. I've added comments to each CI config accordingly in 105ddc005e03ef1d3240eed82628a6c1d43d811d.